shared/profile/folders/container.js

```diff
@@ -0,0 +1,120 @@
+            )
+            .map(tlfName => ({
+              openInFilesTab: () =>
+                dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/public/${tlfName}`)),
+              isPublic: true,
+              isSelf: false,
+              text: `public/${tlfName}`,
+            })),
+          // self private folder
+          {
+            openInFilesTab: () =>
+              dispatchProps._openInFilesTab(
+                FsTypes.stringToPath(`/keybase/private/${stateProps._ownUsername}`)
+              ),
+            isPublic: false,
+            isSelf: true,
+            text: `private/${stateProps._ownUsername}`,
+          },
+          // all other favorited private folders
+          ...stateProps._tlfs.private
+            .keySeq()
+            .filter(tlfName => tlfName !== stateProps._ownUsername)
+            .map(tlfName => ({
+              openInFilesTab: () =>
+                dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/private/${tlfName}`)),
+              isPublic: false,
+              isSelf: false,
+              text: `private/${tlfName}`,
+            })),
+        ]
+      : [
+          // this's profile's public folder
+          {
+            openInFilesTab: () =>
+              dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/public/${profileUsername}`)),
+            isPublic: true,
+            isSelf: true,
+            text: `public/${profileUsername}`,
+          },
+          // other favorited public folders where the profile user is a member
+          ...stateProps._tlfs.public
+            .keySeq()
+            .filter(tlfName => tlfName !== profileUsername && tlfName.split(/[,#]/).includes(profileUsername))
+            .map(tlfName => ({
+              openInFilesTab: () =>
+                dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/public/${tlfName}`)),
+              isPublic: true,
+              isSelf: false,
+              text: `public/${tlfName}`,
+            })),
+          // all favorited private folders where the profile user is a member
+          ...stateProps._tlfs.private
+            .keySeq()
+            .filter(tlfName => tlfName.split(/[,#]/).includes(profileUsername))
+            .map(tlfName => ({
+              openInFilesTab: () =>
+                dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/private/${tlfName}`)),
+              isPublic: false,
+              isSelf: false,
+              text: `private/${tlfName}`,
+            })),
+        ],
+  loadTlfs: dispatchProps.loadTlfs,
+})
+
+const TlfsLoadingHoc = (ComposedComponent: React.ComponentType<any>) =>
+  class extends React.PureComponent<{loadTlfs: () => void}> {
+    componentDidMount() {
+      this.props.loadTlfs()
+    }
+    render() {
+      return <ComposedComponent {...this.props} />
+    }
+  }
+
+export default compose(
+  flags.admin
```

Comment:
why have this be only admin

Reply:
@cecileboucheron thought we should let admins play with it first, get a feel about it, before releasing to the public, as it might look bad and we might want to change layout. Should this use a separate flags as well?